### PR TITLE
[VL] rebase to velox 2023-11-14

### PR DESCRIPTION
**Pull request by FelixYBW**
rebase velox to 2023-11-14

### CODE REVIEW

**Review by PHILO-HE — COMMENTED** [review]
Looks good! Please change repo link to oap/velox before merging. Thanks!

File: ep/build-velox/src/get_velox.sh [hunk 1]
```diff
@@ -16,8 +16,8 @@
 
 set -exu
 
-VELOX_REPO=https://github.com/oap-project/velox.git
-VELOX_BRANCH=2023-11-14
+VELOX_REPO=https://github.com/felixybw/velox.git
+VELOX_BRANCH=2023-11-15
```
Comment (zhouyuan) [inline ep/build-velox/src/get_velox.sh:20]: @FelixYBW can we use the `update` branch? this will makes it difficult make pull requests to oap/velox

**Review by JkSelf — APPROVED** [review]
LGTM. Thanks.

**Pull request merged**